src/localize/languages/pt_br.json

```
@@ -22,13 +22,13 @@
       "live_provider": "Provedor de visualização ao vivo para esta câmera",
       "live_providers": {
         "auto": "Automatico",
-        "frigate": "Frigate",
+        "ha": "Home Assistant (ou seja HLS, LL-HLS, WebRTC nativo)",
```

```suggestion
        "ha": "Home Assistant (ou seja, HLS, LL-HLS ou WebRTC nativo)",
```

But, I think it could also be simplified to:

```suggestion
        "ha": "Home Assistant (HLS, LL-HLS ou WebRTC nativo)",
```